flask/app.py

```diff
@@ -525,10 +531,14 @@ def __init__(self, import_name, static_path=None, static_url_path=None,
         # while the server is running (usually happens during development)
```

Comment:
(Not related to this PR, but in the line above in this comment it says "Do that even if the folder does not exist", yet the the code that follows actually only runs `if self.has_static_folder`. Is the comment wrong or just confusing?)